### PR TITLE
NMS-13472: update table formatting in docs

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13472

### CODE REVIEW

**Comment by Bonrob2** [comment]
Next batch of files. Note that the Enhanced Links and Transparent Bridge Discovery files could use some TLC. I updated basic typos and clarified what I could, but we should probably create separate Jiras to go over them in more detail.

File: docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc [hunk 1]
```diff
@@ -33,36 +33,37 @@ The `building` asset fields for node 1 and node 2 are set to 21 (both nodes are
...
-Keys beginning with `asset-`  come from the corresponding asset table entry for the given node (If defined).
+Keys beginning with `parent-` come from the node table entry of the designated parent node (if defined).
+Keys beginning with `asset-`  come from the corresponding asset table entry for the given node (if defined).
 
 [options="autowidth"]
 |===
-| node-nodelabel | node-nodeid | node-foreignsource | node-foreignid | node-nodesysname 
-| node-nodesyslocation | node-operatingsystem | node-categories| | 
-| parent-nodelabel | parent-nodeid | parent-foreignsource | parent-foreignid | 
+| node-nodelabel | node-nodeid | node-foreignsource | node-foreignid | node-nodesysname
```
Comment (mmahacek) [inline docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc:64]:
```suggestion
5+| *Node fields*
| node-nodelabel | node-nodeid | node-foreignsource | node-foreignid | node-nodesysname
```

File: docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc [hunk 2]
```diff
@@ -33,36 +33,37 @@ The `building` asset fields for node 1 and node 2 are set to 21 (both nodes are
...
+| node-nodesyslocation | node-operatingsystem | node-categories| |
+| parent-nodelabel | parent-nodeid | parent-foreignsource | parent-foreignid |
```
Comment (mmahacek) [inline docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc:66]:
```suggestion
5+| *Parent node fields*
| parent-nodelabel | parent-nodeid | parent-foreignsource | parent-foreignid |
5+| *Node asset fields*
```

File: docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc [hunk 3]
```diff
@@ -133,7 +138,7 @@ The Asset Topology Provider persists GraphML graphs alongside any other GraphML
 <opennms home>/etc/graphml
```
Comment (mmahacek) [inline docs/modules/operation/pages/asset-topology/graphml-asset-topology-provider.adoc:138]:
```suggestion
$OPENNMS_HOME/etc/graphml
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 4]
```diff
@@ -73,157 +73,162 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
...
+[cols="1,1,3"]
 |===
 | Field name  | Field type         | Field description
-| `<label>`   | `java.lang.Double` | Each `Source` defined as `transient=false` can be used as a field.
-                                                        The name of the field is the `label`, e.g., `IfInOctets`
-| `timestamp` | `java.util.Date`   | The timestamp of the sample.
-| `step`      | `java.lang.Long`   | The `Step` size of the `Response`. Returns the same value for all rows.
-| `start`     | `java.lang.Long`   | The `Start` timestamp in milliseconds of the `Resopnse`. Returns the same value for all rows.
-| `end`       | `java.lang.Long`   | The `End` timestamp in milliseconds of the `Response`. Returns the same value for all rows.
+| <label>   | java.lang.Double | Each source defined as `transient=false` can be used as a field.
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:152]: Should this have the < > surrounding the name?

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 5]
```diff
@@ -73,157 +73,162 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
...
 [[ga-database-reports-node-source-descriptor]]
 === Node source descriptor use
 
-A node is addressed by a `node source descriptor`.
-The `node source descriptor` references the node either via the `foreign source` and `foreign id` or by the `node id`.
+A node is addressed by a node source descriptor.
+The node source descriptor references the node either via the foreign source and foreign id or by the node id.
 
-If `store by foreign source` is enabled addressing the node only via `foreign source` and `foreign id` is possible.
+If `store by foreign source` is enabled, is is possible to address the node only via `foreign source` and `foreign id`.
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:223]:
```suggestion
If `storeByForeignSource` is enabled, it is only possible to address the node via foreign source and foreign id.
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 6]
```diff
@@ -73,157 +73,162 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
...
 
-To make report creation easier, there is a helper method to create the `node source descriptor`.
+To make report creation easier, there is a helper method to create the node source descriptor.
 
-NOTE: For more information about `store by foreign source`, please see link:https://wiki.opennms.org/wiki/ForeignSource/foreignId_Data_Storage_How-To[our Wiki].
+NOTE: For more information about store by foreign source, see link:https://wiki.opennms.org/wiki/ForeignSource/foreignId_Data_Storage_How-To[our Wiki].
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:227]: Should not have wiki links - and the referenced wiki page has also been moved to https://opennms.discourse.group/t/storing-data-with-foreign-sources/2057, though the content of this is something that should exist somewhere in the docs as a best practice.

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 7]
```diff
@@ -235,16 +240,16 @@ The following example shows the use of that helper.
...
 === Interface descriptor use
 
-An `interfaceSnmp` is addressed with the exact `interface descriptor`.
-To allow easy access to the `interface descriptor` we provide a helper tool.
+An interfaceSnmp is addressed with the exact interface descriptor.
+To allow easy access to the interface descriptor, we provide a helper tool.
 The following example shows the use of that helper.
 
-.jrxml report snippet to visualize the use of the `interface descriptor`
+.jrxml report snippet to visualize the use of the interface descriptor
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:252]: There are two `\` in the following source block that can be removed.

**Review by mmahacek — COMMENTED** [review]
There were a couple larger changes outside of the review area that I wasn't able to comment on, and I pushed a couple commits to this branch to address them.

File: docs/modules/operation/pages/database-reports/database.adoc (hunk 4 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc:152]: nope. thanks!

File: docs/modules/operation/pages/database-reports/database.adoc (hunk 6 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc:227]: Updated and created Jira: https://issues.opennms.org/browse/NMS-13482

File: docs/modules/operation/pages/database-reports/database.adoc (hunk 7 above)
Comment (Bonrob2) [reply docs/modules/operation/pages/database-reports/database.adoc:252]: I see 5. Can I remove all of them?

File: docs/modules/operation/pages/enlinkd/layer-2/bridge-discovery.adoc [hunk 8]
```diff
@@ -1,136 +1,142 @@
...
+Each VLAN has its own bridge forwarding table and its own spanning tree.
+To discover a bridge topology the algorithm has to be run against every bridge and every vlan.
+Actually the discovery is run only against the main VLAN.
 
-Bridge Domains provide no information about layer 3 but only a layer 2 two map of the Broadcast Domains.
-While Bridge/Switch are identified by the fact that are OpenNMS Nodes to map mac to Nodes where possible the IpNetToMedia table is needed.
-In this manned we are able to associate to mac address the corresponding ip address and then the associated node.
-The Bridge Topology Updater put together the information stored into bridge domains with the ipnettomedia data and provide Bridge OnmsTopology.
+Bridge domains provide no information about layer 3 but only a layer 2 map of the broadcast domains.
+While bridge/switch is identified by the fact that are OpenNMS nodes to map MAC to Nodes where possible the IpNetToMedia table is needed.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/enlinkd/layer-2/bridge-discovery.adoc:26]: Same here. Maybe @indigo423  knows the intent?

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 9]
```diff
@@ -73,178 +73,183 @@ In addition, create a new entry in the `$\{OPENNMS_HOME}/etc/jasper-reports.xml`
...
 
 The following example shows the use of that helper.
 
-.jrxml report snippet to visualize the use of the `node source descriptor`.
+.jrxml report snippet to visualize the use of the node source descriptor.
 [source, xml]
 ------
 <parameter name="nodeResourceDescriptor" class="java.lang.String" isForPrompting="false">
-  <defaultValueExpression><![CDATA[org.opennms.netmgt.jasper.helper.MeasurementsHelper.getNodeOrNodeSourceDescriptor(String.valueOf($P\{nodeid}), $P\{foreignsource}, $P\{foreignid})]]></defaultValueExpression>
+  <defaultValueExpression><![CDATA[org.opennms.netmgt.jasper.helper.MeasurementsHelper.getNodeOrNodeSourceDescriptor(String.valueOf($Pnodeid), $Pforeignsource, $Pforeignid)]]></defaultValueExpression>
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:235]:
```suggestion
  <defaultValueExpression><![CDATA[org.opennms.netmgt.jasper.helper.MeasurementsHelper.getNodeOrNodeSourceDescriptor(String.valueOf($P{nodeid}), $Pforeignsource, $P{foreignid})]]></defaultValueExpression>
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 10]
```diff
@@ -253,32 +258,32 @@ The following example shows the use of that helper.
 </parameter>
 <queryString language="Measurement">
   <![CDATA[<query-request step="300000" start="$P{startDateTime}" end="$P{endDateTime}" maxrows="2000">
-<source aggregation="AVERAGE" label="IfInOctets" attribute="ifHCInOctets" transient="false" resourceId="node[$P\{nodeid}].interfaceSnmp[$P{interface}]"/>
-<source aggregation="AVERAGE" label="IfOutOctets" attribute="ifHCOutOctets" transient="false" resourceId="node[$P\{nodeid}].interfaceSnmp[$P{interface}]"/>
+<source aggregation="AVERAGE" label="IfInOctets" attribute="ifHCInOctets" transient="false" resourceId="node[$Pnodeid].interfaceSnmp[$P{interface}]"/>
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:261]:
```suggestion
<source aggregation="AVERAGE" label="IfInOctets" attribute="ifHCInOctets" transient="false" resourceId="node[$P{nodeid}].interfaceSnmp[$P{interface}]"/>
```

File: docs/modules/operation/pages/database-reports/database.adoc [hunk 11]
```diff
@@ -253,32 +258,32 @@ The following example shows the use of that helper.
...
+<source aggregation="AVERAGE" label="IfOutOctets" attribute="ifHCOutOctets" transient="false" resourceId="node[$Pnodeid].interfaceSnmp[$P{interface}]"/>
```
Comment (mmahacek) [inline docs/modules/operation/pages/database-reports/database.adoc:262]:
```suggestion
<source aggregation="AVERAGE" label="IfOutOctets" attribute="ifHCOutOctets" transient="false" resourceId="node[$P{nodeid}].interfaceSnmp[$P{interface}]"/>
```

File: docs/modules/operation/pages/enlinkd/layer-2/bridge-discovery.adoc [hunk 12]
```diff
@@ -1,136 +1,142 @@
...
+In this manner we can associate a MAC address with the corresponding IP address and then the associated node.
```
Comment (mmahacek) [inline docs/modules/operation/pages/enlinkd/layer-2/bridge-discovery.adoc:27]:
```suggestion
In this manner, we can associate a MAC address with the corresponding IP address and then the associated node.
```

**Review by mmahacek — APPROVED** [review]
A couple typos, and the rest looks good.

**Pull request merged**